x/auth/keeper/account.go

```diff
@@ -1,6 +1,7 @@
 package keeper
 
 import (
+	store2 "github.com/cosmos/cosmos-sdk/store"
```

Comment:
This is in a lot of places, let's leave this one as is and rename the store variable to something like `st`? (`store := ctx.KVStore(ak.storeKey)`). Just to make it easier to read.

```suggestion
	"github.com/cosmos/cosmos-sdk/store"
```

Reply:
+1

Reply:
Re-opening this one, wdyt? @likhita-809